zaraz.go

```diff
@@ -0,0 +1,234 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/goccy/go-json"
+)
+
+type ZarazConfig = map[string]interface{}
+
+type ZarazConfigResponse struct {
+	Result ZarazConfig `json:"result"`
+	Response
+}
+
+type ZarazWorkflowResponse struct {
+	Result string `json:"result"`
+	Response
+}
+
+type ZarazPublishResponse struct {
+	Result string `json:"result"`
+	Response
+}
+
+type UpdateZarazConfigParams = map[string]interface{}
```

Comment:
this should be a struct to prevent any unknown keys from being pushed into the parameters.